examples/lib/intermediateHtmlStepUi.js

```diff
@@ -118,9 +118,9 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
       sortField: 'text'
     });
     $(step.ui.querySelector('.inserDiv .add-step-btn')).prop('disabled', true);
-
-    insertStepSelect.append('<option value="none" disabled selected>More modules...</option>');
-    $(step.ui.querySelector('.insertDiv .radio-group .radio')).on("click", function () {
+    
+    insertStepSelect.append('<option value="" disabled selected>Select a Module</option>');
+    $('#insertStep .radio-group .radio').on("click", function () {
```

Comment:
```suggestion
    $(step.ui.querySelector('.insertDiv .radio-group .radio')).on("click", function () {
```

You can merge these suggested right away in the broswer and your PR will be good to go.